userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -232,6 +234,8 @@ Product Category:: A type of product. Possible values are <<addon, ADDON>>, <<ba
 
 == R
 
+Reference Time:: UTC Time associated with an account that is used to to convert a `LocalDate` to a `DateTime` and vice-versa.
```

Comment:
Can you point to your scenarios below for an example to follow?

Reply:
Yes, will do, let's leave this as it is for now, once we finalize the examples in the other document, I'll point to it.